spec/requests/automation_requests_spec.rb

```diff
@@ -296,15 +296,11 @@
     end
 
     context "SubResource#cancel" do
-      let(:resource_1_response) { {"success" => false, "message" => "Cancel operation is not supported for AutomationTask"} }
-      let(:resource_2_response) { {"success" => false, "message" => "Cancel operation is not supported for AutomationTask"} }
```

Comment:
Why would these change?  I don't see any changes to Cancel such that we should stop testing these?

Reply:
nvm - didn't realize this was a shared example and the shared example changed.